src/index.html

```diff
@@ -0,0 +1,36 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script defer src="/index.bundle.js"></script>
+</head>
+
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+  <main class="d-flex">
+    <div class="flex-1" id="table-container">
+      <div class="d-flex" id = "table-header">
+        <h2>Recent scores</h2>
+        <button id="refresh-btn">Refresh!</button>
+      </div>
```

Comment:
- [ ] Please add a list of recent scores as shown in the wireframe example. Everything else is great :clap: . Kindly make this change and you are good togo.

![image](https://user-images.githubusercontent.com/64894927/128301918-ed7dd1be-c7de-480a-a4cb-cdeb52212e1e.png)